### DISCUSSION TITLE
hi tsyuay,  do it please!! my last requests before you go on vacation.

### DISCUSSION THREAD

**Discussion by demonster2023** [Tsyuay/FrivReborn#37] [Q&A] [answered]
<img width="903" alt="shape switsher" src="https://user-images.githubusercontent.com/121765699/213450216-68401afb-3740-4abc-9bf4-8a5c015a6e37.png">

**Comment by demonster2023** [comment]
and do it please

<img width="946" alt="game 2" src="https://user-images.githubusercontent.com/121765699/213455043-5b9fa0af-fa19-490a-a3ef-0bd50b9fbdb7.png">

**Comment by Tsyuay** [comment]
Sure

**Comment by Tsyuay** [comment, marked as answer]
done, but sadly I couldn't make it perfect with the second game you wanted me to fix, I tried all I could EDIT: nevermind, it should be perfect now, but the size may not be same
